{
  "code": null,
  "results": {
    "assert next_Perfect_Square(35) == 36": {
      "result": false,
      "time": -1,
      "error": "GenerationError"
    },
    "assert next_Perfect_Square(6) == 9": {
      "result": false,
      "time": -1,
      "error": "GenerationError"
    },
    "assert next_Perfect_Square(9) == 16": {
      "result": false,
      "time": -1,
      "error": "GenerationError"
    }
  },
  "passed": false,
  "accuracy": 0.0,
  "general_error": "<class 'AssertionError'>",
  "has_testcase_error": true
}